internal/util/flowgraph/flow_graph.go

```diff
@@ -20,16 +20,27 @@ import (
 	"context"
 	"errors"
 	"sync"
+	"time"
+
+	"github.com/milvus-io/milvus/internal/log"
+	"github.com/milvus-io/milvus/internal/util/lock"
+	"go.uber.org/zap"
+)
+
+const (
+	TryLockNodeInterval = 1 * time.Second
+	TryLockRetryCount   = 5
 )
 
 // Flow Graph is no longer a graph rather than a simple pipeline, this simplified our code and increase recovery speed - xiaofan.
 
 // TimeTickedFlowGraph flowgraph with input from tt msg stream
 type TimeTickedFlowGraph struct {
-	nodeCtx   map[NodeName]*nodeCtx
-	stopOnce  sync.Once
-	startOnce sync.Once
-	closeWg   *sync.WaitGroup
+	nodeCtx      map[NodeName]*nodeCtx
+	nodeSequence []string
```

Comment:
Please check other flowgraph operations by nodeCtx should follow the order of the node in nodeSequence.